### PR TITLE
Added new way of publishing test results

### PR DESCRIPTION
**Pull request by aggarwav**

### CODE REVIEW

File: risk_cd.md [hunk 1]
```diff
@@ -16,32 +16,40 @@ lastupdated: "2017-07-31"
 
 You can instrument pipelines for {{site.data.keyword.contdelivery_full}} to use {{site.data.keyword.DRA_short}}' Deployment Risk analysis capabilities. For more information about Continuous Delivery pipelines, see [the official documentation](../ContinuousDelivery/pipeline_working.html).
 
-After you add {{site.data.keyword.DRA_short}} to your toolchain, define policies for it. Then, you can instrument your pipelines to send data to {{site.data.keyword.DRA_short}} and enforce those policies.
+After you add {{site.data.keyword.DRA_short}} to your toolchain, you can instrument your pipelines to publish build, deploy and test records to {{site.data.keyword.DRA_short}}.
```
Comment (maxbridges) [inline risk_cd.md]: Making some notes for my editing...
deploy -> deployment
`conref` -> it

File: risk_cd.md [hunk 2]
```diff
@@ -16,32 +16,40 @@ lastupdated: "2017-07-31"
...
+
+Once your pipeline starts publishing data to {{site.data.keyword.DRA_short}}, define risk policies. More information on defining risk policies is here (https://console.bluemix.net/docs/services/DevOpsInsights/risk_policies.html#policies_and_rules).
```
Comment (maxbridges) [inline risk_cd.md]: * Change docs URL to relative
* Use IBM-recommended format for link out

File: risk_cd.md [hunk 3]
```diff
@@ -16,32 +16,40 @@ lastupdated: "2017-07-31"
 
 You can instrument pipelines for {{site.data.keyword.contdelivery_full}} to use {{site.data.keyword.DRA_short}}' Deployment Risk analysis capabilities. For more information about Continuous Delivery pipelines, see [the official documentation](../ContinuousDelivery/pipeline_working.html).
 
-After you add {{site.data.keyword.DRA_short}} to your toolchain, define policies for it. Then, you can instrument your pipelines to send data to {{site.data.keyword.DRA_short}} and enforce those policies.
+After you add {{site.data.keyword.DRA_short}} to your toolchain, you can instrument your pipelines to publish build, deploy and test records to {{site.data.keyword.DRA_short}}.
+
+Once your pipeline starts publishing data to {{site.data.keyword.DRA_short}}, define risk policies. More information on defining risk policies is here (https://console.bluemix.net/docs/services/DevOpsInsights/risk_policies.html#policies_and_rules).  
+
+Once these risk policices have been defined, they can be enforced by use of Gate jobs in your pipeline.
```
Comment (maxbridges) [inline risk_cd.md]: "Once you define risk policies, you can enforce them by adding gate jobs to your pipeline."

File: risk_cd.md [hunk 4]
```diff
@@ -16,32 +16,40 @@ lastupdated: "2017-07-31"
...
+
+
 
 ## Preparing pipeline stages and jobs
 {: #integrate_pipeline}
 
-To get started, you need to instrument your pipeline to communicate with {{site.data.keyword.DRA_short}}. You do this by defining specific environment variables for all of your pipeline jobs that build, test, and deploy code.
+To get started, you need to instrument your pipeline to communicate with {{site.data.keyword.DRA_short}}. You do this by defining specific environment variables for all of your pipeline jobs that build, test, deploy code or that enforce risk policies by using Gate job type.
```
Comment (maxbridges) [inline risk_cd.md]: * Gate -> gate
* Serial comma

File: risk_cd.md [hunk 5]
```diff
@@ -16,32 +16,40 @@ lastupdated: "2017-07-31"
...
 
-For build jobs in your pipeline, set environment variables for an application name and build prefix. An example script would include these commands:
+For the last build job in a stage, set environment variables for an application name and build prefix. An example script would include these commands:
 
 ```
 export LOGICAL_APP_NAME="SampleApp"
 export BUILD_PREFIX="master"
 ```
 
+When this build job completes, the Pipeline would publish a message to {{site.data.keyword.DRA_short}} that a build for an is completed.
```
Comment (maxbridges) [inline risk_cd.md]: * Pipeline -> pipeline
* "that a SampleApp build is completed"

File: risk_cd.md [hunk 6]
```diff
@@ -52,29 +60,31 @@ export BUILD_PREFIX="master"
 export LOGICAL_ENV_NAME="Production"
 ```
 
-### Configuring test jobs
+When your deployment job ends, the Pipeline would publish a message to {{site.data.keyword.DRA_short}} that the specified app and build have been deployed to an environment.
```
Comment (maxbridges) [inline risk_cd.md]: Same as above.

File: risk_cd.md [hunk 7]
```diff
@@ -52,29 +60,31 @@ export BUILD_PREFIX="master"
...
-If the job publishes functional verification test (FVT) results, you must also set the logical environment name to wherever those tests run.
+If the job generates functional verification test (FVT) results, you must also set the logical environment name to wherever those tests run.
 
 An example script would include these commands:
 
 ```
 export LOGICAL_APP_NAME="SampleApp"
 export BUILD_PREFIX="master"
+
+#LOGICAL_ENV_NAME only needed if publishing FVT results
```
Comment (maxbridges) [inline risk_cd.md]: "The `LOGICAL_ENV_NAME` variable is only needed when publishing FVT results."

File: risk_cd.md [hunk 8]
```diff
@@ -52,29 +60,31 @@ export BUILD_PREFIX="master"
...
 export LOGICAL_ENV_NAME="Production"
 ```
 
 Make sure that application names and environments match where appropriate. For example, you would want a production test job that runs against a production deployment to have identical `LOGICAL_ENV_NAME` values.
 
 
-## Adding test jobs
+## Publishing test data to DevOps Insights using Advanced Tester job type
```
Comment (maxbridges) [inline risk_cd.md]: Will probably break this out differently but the content should stay largely the same.

**Comment by maxbridges** [comment]
I've updated staging and prod with edits. Feel free to make another PR whenever you have feedback/input/additions.

**Comment by aggarwav** [comment]
Thanks @maxbridges

**Pull request merged**